File: nomad-samples/common/js/account-registration/account/resend-code.js
import * as prjConstants from "../constants/project-constants.js";
import apiExceptionHandler from "../exceptions/api-exception-handler.js";

export default async function resendCode(EMAIL) 
{
		// Create header for the request
    const HEADERS = new Headers();
    HEADERS.append("Content-Type", "application/json");
  
  	const BODY = {
        userName: EMAIL
    };
  
  // Post
    const RESPONSE = await fetch(`${prjConstants.PORTAL_API_URL}/account/resend-code`, {
        method: "POST",
        headers: HEADERS,
        body: JSON.stringify(BODY)
    }).catch((exception) => {
        throw exception;
    });

    // Check for success
    if (RESPONSE.ok) {
      return(RESPONSE.status);
    }
		
  	// There was an error
    await apiExceptionHandler(RESPONSE, `Resend code failed`)
}

File: nomad-samples/common/py/account_registration/account/resend_code.py
from constants.project_constants import *
from exceptions.api_exception_handler import *

import json
import requests

def resend_code(EMAIL: str) -> dict:
    if not EMAIL:
        raise Exception("Email: The email is invalid")

    API_URL = PORTAL_API_URL + "/account/resend-code"

    HEADERS = {
      "Content-Type": "application/json"
    }
    
    # replace username and password with your username and password
    BODY = {
      "userName": EMAIL
    }

    try:
        RESPONSE = requests.post(API_URL, headers=HEADERS, data=json.dumps(BODY))
        if not RESPONSE.ok:
          raise Exception()
    except:
      api_exception_handler(RESPONSE, "Resend code failed")
            
